### PR TITLE
durability fambrik thing

### PR DESCRIPTION
**Pull request by MoriyaShiine**

### CODE REVIEW

**Comment by CammiePone** [comment]
that branch name is horrific

**Comment by williewillus** [comment]
you have build failures.

**Comment by MoriyaShiine** [comment]
my dad owns microsoft I can get you banned from fortnite if you don't review

File: src/main/java/vazkii/botania/api/item/IDurabilityExtension.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * Botania is Open Source and distributed under the
+ * Botania License: http://botaniamod.net/license.php
+ */
+package vazkii.botania.api.item;
+
+import net.minecraft.item.ItemStack;
+
+/**
+ * An Item that has this component can have its durability bar modified.
```
Comment (williewillus) [inline src/main/java/vazkii/botania/api/item/IDurabilityExtension.java:14]: component?

Comment (MoriyaShiine) [reply src/main/java/vazkii/botania/api/item/IDurabilityExtension.java]: Fixed

**Comment by williewillus** [comment]
move it out of the api also, no one needs to be using this besides botania itself

**Comment by MoriyaShiine** [comment]
Done

**Pull request merged**